src/locales/swedish.js

```diff
@@ -0,0 +1,386 @@
+  },
+  ROLES: {
+    MY_ROLE_NOT_HIGH_ENOUGH: "Min roll är inte tillräckligt hög än **{role}** roll!",
+    MY_ROLE_MUST_BE_HIGHER:
+      "Min roll måste vara högre än **{member}** högsta roll!",
+    YOUR_ROLE_MUST_BE_HIGHER: "Din roll måste vara högre än **{role}** roll!",
+    PROVIDE_ROLE: "Vänligen ange en roll",
+    ALREADY_HAS_ROLE: "Den medlemmen har redan den rollen",
+    ADDED_ROLE_TO: "har lagts till **{role}** till {member}",
+  },
+  IMAGE: {
+    CLICK_TO_VIEW: "[Klicka här om bilden inte kunde laddas.]",
+    CLYDE: "Clyde",
+    CUDDLES: "kramar med",
+    FEEDED: "matas",
+    HUGGED: "kramade",
+    PATTED: "klappade",
+    KISSED: "kysste",
+    POKED: "stötte",
+    SLAPPED: "slog",
+  },
+  ANIMAL: {
+    CAT_FACT: "Kattfakta",
+    DOG_FACT: "Hundfakta",
+    SNAIL_FACT: "Snigelfakta",
+    COW: "Ko",
+  },
+  BOT_OWNER: {
+    SHUTDOWN: "Bot stängs av",
+    EVAL: "Eval-kommando",
+    EVAL_TYPE: "Typ",
+    EVAL_INPUT: "Inmatning",
+    EVAL_OUTPUT: "Produktion",
+    UPDATE_NICKNAME: "Uppdaterad av bot-ägaren",
+    UPDATED_NICKNAME: "Smeknamnet har uppdaterats till **{nickname}**",
+    PROVIDE_TYPE: "Vänligen ange en typ",
+    CANNOT_BL_OWNER: "Det går inte att svartlista ägaren",
+    NOT_BLD: "Användaren är inte svartlistad",
+    ALREADY_BLD: "{member} är redan svartlistad",
+    NOT_OPTION: "**{type}** är inte ett alternativ",
+    BLACKLISTED: "svartlistad",
+    UNBLACKLISTED: "osvartlistad",
+    BLACKLISTED_SUCCESS: "{member} var {type}",
+    BLD_STATUS: "Svartlistans status",
+  },
+  LEVELS: {
+    XP: "XP",
+    LEVEL: "Nivå",
+    LEADERBOARD: "Leaderboard",
+    MEMBER_IS_LEVEL: "{member} är nivå **{level}** med **{user_xp}XP**",
+    RESET_CONF: "Reset all XP? y/n",
+    RESET_SUCCESS: "Återställ allas xp ",
+    RESET_CANCEL: "resetxp avbröts",
+    PROVIDE_AMOUNT: "Vänligen ange ett belopp",
+    PROVIDE_VALID_NR: "Ange ett giltigt nummer",
+    GIVE_XP_SUCCESS: "SFramgångsrikt gav **{member}** **{amount}**XP",
+  },
+  MUSIC: {
+    MUST_BE_IN_VC: "Du måste vara i en röstkanal",
+    NO_QUEUE: "Det finns inga låtar för närvarande",
+    QUEUE_CLEARED: "Kön rensades",
+    QUEUE: "Musik kö",
+    BETWEEN_0_100: "Volymen måste vara mellan 0 och 100",
+    VOL_SUCCESS: "Ställ in volymen till {vol}%",
+    PLAYING: "Spelar",
+    PAUSED: "Paused",
+    DURATION: "Varaktighet",
+    UPLOADED_BY: "uppladdad av",
+    UPLOADED_AT: "Uppladdat kl",
+    VIEWS: "Visningar",
+    LIKES: "Gillar",
+    DISLIKES: "Ogillar inte",
+    SECONDS: "Seconds",
+    VOLUME: "Volume",
+    PROVIDE_SEARCH: "Ange en sökfråga",
+    ADDED_TO_QUEUE: "Sång har lagts till i kön | {songs} låtar i kö",
+    NOW_PLAYING: "Nu spelas:",
+    REQUESTED_BY: "Efterfrågad av",
+    NO_PERMS: "Jag har inte rätt behörighet för den röstkanalen!",
+    NOT_SAME_VC: "Du måste vara i samma röstchatt som botten",
+    ALREADY_IN_VC: "Jag är redan i den röstchatten",
```

Comment:
There are 2 lines missing here: 

`MUST_BE_SAME_VC: "You must be in the same voice chat with the bot",`
`NO_RESULTS: "No songs were found",`